.editorconfig

```diff
@@ -0,0 +1,9 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+insert_final_newline = true
+indent_style = space
+indent_size = 4
```

Comment:
I'd prefer to remove the indentation changes now instead of merging them into main and having to undo them.

Reply:
at this point it will be unnecessary work. Most of these files are new and created in 4, so there's no undo-ing.
 I'm hoping to make the case for 4 instead of 2. :)